File: Java/001-multiples-of-3-and-5.java
class Solution
{
  public static long multiples(long n)
  {
    long sum = 0 ;
      
      for(long i = 1 ; i < n ; i++) 
    {
    	if((i % 3 == 0 || i % 5 == 0))
        sum+=i;
    }
    
    return sum;
  }
	public static void main(String[] args) 
  {
      long ans = multiples(1000);
      System.out.println(ans);
  }
}

File: CSharp/001-multiples-of-3-and-5.cs
using System;

namespace ProjectEuler
{
    class Program
    {
        static void Main(string[] args)
        {
            var limit = 1000;
            var sum = SumMultipleThreeOrFive(limit);

            Console.WriteLine($"The sum of multiples below {limit} is {sum}");
        }

        static long SumMultipleThreeOrFive(int limit)
        {
            long sum = 0;

            for (int i = 0; i < limit; i++)
            {
                if (i % 3 == 0 || i % 5 == 0)
                    sum += i;
            }

            return sum;
        }
    }
}
